sdks/python/apache_beam/runners/worker/statesampler.pyx

```diff
@@ -151,6 +157,30 @@ cdef class StateSampler(object):
           nsecs_ptr = &(<ScopedState>PyList_GET_ITEM(
               self.scoped_states_by_index, self.current_state_index)).nsecs
           nsecs_ptr[0] += elapsed_nsecs
+
+          active_nsecs_ptr = &(<ScopedState>PyList_GET_ITEM(
```

Comment:
<!--new_thread; commit:c0086ba69497a5cbd3269f1d691831ca78563dcc; resolved:0-->
Can you clearly delimit the portion of this code that is dedicated to state time sampling and active time sampling?